### PR TITLE
Remove unnnecessary examples of TableRegistry usage.

### PR DESCRIPTION
**Pull request by ADmad**
It promotes bad practice.

### CODE REVIEW

**Comment by saeideng** [comment]
you can clear it in `orm/table-objects.rst` too
but you have to mention one of available ways
```php
use Cake\ORM\Locator\TableLocator;

$locator = new TableLocator();
$articles = $locator->get('Articles');
```
```php
use Cake\ORM\Locator\LocatorAwareTrait;

$articles = $this->getTableLocator()->get('Articles');
```

File: en/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -1039,10 +1014,6 @@ Some other examples of the collection methods being used with result sets are::
     });
 
     // Create an associative array from result properties
-    // Prior to 3.6.0
-    $articles = TableRegistry::get('Articles');
-
-    $articles = TableRegistry::getTableLocator()->get('Articles');
     $results = $articles->find()->contain(['Authors'])->all();
```
Comment (dereuromark) [inline en/orm/retrieving-data-and-resultsets.rst]: But shouldnt those then be class props?

    $this->Articles->find()

etc
As you probably loaded them via trait.

Comment (dereuromark) [reply en/orm/retrieving-data-and-resultsets.rst]: Also:
We either expect them to know how to load those tables, or ideally add the call:

    $this->loadModel('Articles');

Especially, if the current controller is not necessarily expected to have this as default model.

**Comment by ADmad** [comment]
There are already numerous other examples  of this page which directly begin with `$articles->`.

**Pull request merged**